cmd/wnode-status/config.go

```diff
@@ -16,98 +45,66 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 		return nil, err
 	}
 
-	// TODO(divan): move this logic into params package?
-	if *nodeKeyFile != "" {
```

Comment:
Why? We don't need this?

Reply:
I haven't seen any usage of it. We can bring it back when we figure out what it is useful for.

Reply:
Ok. In any case we can revert it if needed.